src/Strategy/Node/YarnV2/Resolvers.hs

```diff
@@ -47,12 +48,16 @@ data Resolver = Resolver
   -- ^ Convert this locator to a yarn package
   }
 
+-- Default Yarn Protocols can be found at https://yarnpkg.com/features/protocols.
 data Package
   = WorkspacePackage Text -- relative reference to a directory. not quite a Path Rel Dir because it may contain '..'
   | NpmPackage (Maybe Text) Text Text -- scope, package, version
   | GitPackage Text Text -- url, commit
   | TarPackage Text -- url
   | FilePackage Text
+  | -- LibPackages are a custom protocol supported by a user.
+    -- TODO: Support all custom protocols out of the box.
```

Comment:
[nit] TODO comments on their own are basically never actioned and are largely just noise since we have no TODO triage or audit process. Can we create a ticket for this instead?

Reply:
Will do, I definitely don't disagree with that

Reply:
https://fossa.atlassian.net/browse/ANE-751